analyses/oncoprint-landscape/01-plot-oncoprint.R

```diff
@@ -177,6 +178,50 @@ metadata <- metadata %>%
   # Reorder display_group based on display_order
   dplyr::mutate(display_group = forcats::fct_reorder(display_group, display_order))
 
+# Filter to the metadata associated with the broad histology value, if provided
+if (!is.null(opt$broad_histology)) {
+  
+  if (!opt$broad_histology == "Other CNS") {
+    metadata <- metadata %>%
+      dplyr::filter(broad_histology == opt$broad_histology)
+  } else {
+    metadata <- metadata %>%
+      dplyr::filter(
+        broad_histology %in% c(
+          "Tumors of sellar region",
+          "Neuronal and mixed neuronal-glial tumor",
+          "Tumor of cranial and paraspinal nerves	",
+          "Meningioma",
+          "Mesenchymal non-meningothelial tumor	",
+          "Germ cell tumor",
+          "Choroid plexus tumor	",
+          "Histiocytic tumor",
+          "Tumor of pineal region",
+          "Metastatic tumors",
+          "Other astrocytic tumor",
+          "Lymphoma",
+          "Melanocytic tumor",
+          "Other tumor"
+        )
+      )
+  }
```

Comment:
I think this logic works well for now, but am making a note that this is an area where we might want to take a different approach when we have different genes of interest lists. I don't think it makes sense to make any changes to accommodate #969 just yet!